### DISCUSSION TITLE
Turbulence model and transition models coupled in SU2

### DISCUSSION THREAD

**Discussion by rejirejish** [su2code/SU2#1443] [Q&A]
Which coupled model is reliable to use it in adjoint-based shape optimisation? 
1. SA-BC. 
2. SA-LM.
Two different coupled models are available to use it in the latest version of SU2.

**Comment by pcarruscag** [comment]
SA-BC, LM is disabled at the moment as it was never fully completed.

**Reply by rejirejish** [reply]
Thank you very much for your reply.
Is SA-BC is fully coupled to use it in the discrete adjoint framework..?
